src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -556,24 +551,16 @@ private BigInteger(uint[] value)
             if (!isNegative)
             {
                 // Handle the simple positive value cases where the input is already in sign magnitude
-                if (dwordCount != value.Length)
-                {
-                    _sign = +1;
-                    _bits = new uint[dwordCount];
-                    Array.Copy(value, _bits, dwordCount);
-                }
-                // No trimming is possible.  Assign value directly to _bits.
-                else
-                {
-                    _sign = +1;
-                    _bits = value;
-                }
+                _sign = +1;
```

Comment:
```suggestion
                _sign = 1;
```
? Or is it to put emphasis on the positive value? 
(I see the original code has this too)

Reply:
Yes, I think it was done to highlight that sign of big integer is positive.

Reply:
It might be better to use named constants `Positive` and `Negative` instead?